IFC4x3/Properties/n/NominalThickness_3cTq6sD71BVAf9l72L6lIy/Documentation.md

```diff
@@ -0,0 +1 @@
+The nominal thickness of the pavement. The size information is provided in addition to the shape representation and the geometric parameters used within. In case of inconsistency between the geometric parameters and the size properties, provided in the attached property set, the geometric parameters take precedence. It is regarded as constant thickness of the pavement, unless property NominalThicknessEnd is provided, in which case NominalThickness shall be regarded as the value at the beginning of a transition (usually at the object placement location). Example of such case is a (road) transition segment where the pavement object’s linear placement along an alignment denotes the beginning location.
```

Comment:
Not sure how to fix the last sentence. @AlexBrad1eyCT any suggestion? I would probably just do e.g.

Reply:
>  It is regarded as constant thickness of the pavement, unless property NominalThicknessEnd is provided......

slight rewording to:

> The thickness of the pavement remains constant, unless the property NominalThicknessEnd is provided. In which case NominalThickness is the value at the beginning of a transition (usually at the object placement location). e.g. A (road) transition segment where the pavement object's linear placement along an alignemnt denotes the begining location.